.flake8

```diff
@@ -21,6 +21,7 @@ exclude =
   CVS
   .venv*/
   venv*/
+  */samples/*
```

Comment:
Shouldn't we make sure our samples are formatted correctly, too? Is there a specific thing flake does not like that we want to allow for samples?

Reply:
Yeah it doesn't like the ordering of some imports. I don't think linting is necessary for samples.